Node[] nodes
---
bool node_initiated
